samples/client/echo_api/python/openapi_client/api_client.py

```diff
@@ -386,21 +383,32 @@ def sanitize_for_serialization(self, obj):
             for key, val in obj_dict.items()
         }
 
-    def deserialize(self, response_text, response_type):
+    def deserialize(self, response_text: str, response_type: str, content_type: str):
         """Deserializes response into an object.
 
         :param response: RESTResponse object to be deserialized.
         :param response_type: class literal for
             deserialized object, or string of class name.
+        :param content_type: content type of response.
 
         :return: deserialized object.
         """
 
         # fetch data from response object
-        try:
+        if content_type is None:
+            try:
+                data = json.loads(response_text)
+            except ValueError:
+                data = response_text
+        elif content_type.startswith('application/json'):
             data = json.loads(response_text)
-        except ValueError:
+        elif content_type.startswith('text/plain'):
```

Comment:
To keep consistency:

```suggestion
        elif content_type.startswith("text/plain"):
```